### PR TITLE
Import strings from android-l10n

### PR DESCRIPTION
**Pull request by Pike**
State: mozilla-l10n/android-l10n@4c73402212e4766cf050b520318cb56961c78e2b

More string changes for Fenix. CC @Delphine @pocmo

### CODE REVIEW

**Comment by Delphine** [comment]
Would be nice to include these fixes to release, but if it's too late that should be OK (although we don't know severity of these fixes)

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -161,13 +161,13 @@
     <!-- Label indicating that sync is in progress -->
     <string name="sync_syncing_in_progress">正在同步…</string>
     <!-- Label summary indicating that sync failed. The first parameter is the date stamp showing last time it succeeded -->
-    <string name="sync_failed_summary">同步失败。最后同步：%s</string>
+    <string name="sync_failed_summary">同步失败。上次成功：％s</string>
```
Comment (pocmo) [inline app/src/main/res/values-zh-rCN/strings.xml]: This change looks like it's using the wrong `%` now? @Pike?

Comment (Pike) [reply app/src/main/res/values-zh-rCN/strings.xml]: Good catch, let me fix that upstream and update the PR.

Comment (Pike) [reply app/src/main/res/values-zh-rCN/strings.xml]: FYI, just checked, this is equivalent to a missing placeable. compare-locales does check for params not in en-US, and for mismatching params. It doesn't check for params to be left out, as that's sometimes OK, for example when en-US refers to branding. Some languages always use passive voice there, so the brand name doesn't show up at all.

**Comment by Pike** [comment]
The second commit here fixed the `zh-CN` issue, and coincidentally, the same issue in Japanese two times, too.

File: app/src/main/res/values-ja/strings.xml [hunk 2]
```diff
@@ -574,6 +574,8 @@
     <string name="url_copied">URL をコピーしました</string>
 
 
+    <!-- Sample text for accessiblity font size -->
+    <string name="accessibility_text_size_sample_text">The quick brown fox jumps over the lazy dog. The quick brown fox jumps over the lazy dog. The quick brown fox jumps over the lazy dog. The quick brown fox jumps over the lazy dog.</string>
```
Comment (colintheshots) [inline app/src/main/res/values-ja/strings.xml]: Should the Japanese accessibility text only be in English?

Comment (Pike) [reply app/src/main/res/values-ja/strings.xml]: TBH, I don't know why it's labelled a11y. Is https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/res/layout/layout_percentage_seek_bar.xml#L75 a11y-only?

Comment (flodolo) [reply app/src/main/res/values-ja/strings.xml]: (also, typo in that comment…)

Comment (Pike) [reply app/src/main/res/values-ja/strings.xml]: As always, that's an upstream bug, we copy comments from English.

**Comment by Delphine** [comment]
Is there anything still pending that's blocking us on merging this?

**Pull request merged**